simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/StationState.java

```diff
@@ -69,23 +73,23 @@ public void stopCharging(Integer evseId) {
     }
 
     public Evse getDefaultEvse() {
-        return evses.get(0);
+        return evses.values().stream().findFirst().orElseThrow(() -> new StationException("Default evse not found"));
```

Comment:
Instead of 'default' let's state that 'none' evses were found. Concept of default evse may not be clear enough